kaitai_struct_runtime_csharp.csproj

```diff
@@ -1,12 +1,12 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Configuration Condition=" '$(Configuration)' == '' ">Release</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{8339A750-C407-4CE8-8E31-51CB2EFD3A4B}</ProjectGuid>
     <OutputType>Library</OutputType>
-    <RootNamespace>kaitai_struct_runtime_csharp</RootNamespace>
-    <AssemblyName>kaitai_struct_runtime_csharp</AssemblyName>
+    <RootNamespace>KaitaiStruct</RootNamespace>
+    <AssemblyName>KaitaiStruct.Runtime.CSharp</AssemblyName>
```

Comment:
As far as I understand:

* We use just `Kaitai` namespace everywhere in .NET => probably "root namespace" should be just "Kaitai", although I'm not sure what it affects.
* Assembly name should be probably `Kaitai.Struct.Runtime` or something like that:
  * Judging from Microsoft's recommendations, first component of the assembly name should be organization, then the project, subproject, etc.
  * There would be probably no point to add `CSharp`, as this assembly would work equally well for any .NET language (C#, C++, F#, VB.NET, etc), and it is actually not that important what it is written in originally.